src/app/voice-assistant/voice-assistant-chat/chat-window/chat-window.component.scss

```diff
@@ -116,3 +116,9 @@
     outline: none;
     opacity: 0.3;
 }
+
```

Comment:
why did you add this css?

Reply:
That was not supposed to be part of this PR, so I removed it again.